### PR TITLE
FHAC-570 Updates for NetworkAccessType values

### PR DESCRIPTION
**Pull request by sovanncgi**
Changes to Audit Logging test cases in both standard and passthru folders for both g1 and g0. Updates include:
1. RD Assertion values for ActiveParticipant_Source.@UserID, ActiveParticipant_Source.@NetworkAccessPointID, ActiveParticipant_Dest.@UserID and ActiveParticipant_Dest.@NetworkAccessPointID
2. For all services except PD and DS, a new assertion was added to make sure that ActiveParticipant_Dest.@NetworkAccessPointTypeCode = 1 or 2
3. Depending on the value of ActiveParticipant_Dest.@NetworkAccessPointTypeCode, assertions added to:
    a. If value is 1, assert that the NetworkAccessPointID is a machine name (specifically, "localhost")
    b. If valule is 2, assert that the NetworkAccessPointID is an IP in a valid IP format

### CODE REVIEW

File: Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml [hunk 1]
```diff
@@ -6312,10 +6407,9 @@ if(!parsedXmlMessage.ParticipantObjectIdentification[0].@ParticipantObjectTypeCo
 }]]></script>
         </con:config>
       </con:testStep>
-      <con:testStep type="groovy" name="VerifyMessage-NhinOutbound-InitiatingGateway(1.1)" disabled="true">
```
Comment (TabassumJafri) [inline Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml:6315]: Not sure why this test step was enabled

File: Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml [hunk 2]
```diff
@@ -6761,10 +6879,10 @@ if (!parsedXmlMessage.ParticipantObjectIdentification[0].@ParticipantObjectTypeC
 }]]></script>
         </con:config>
       </con:testStep>
-      <con:testStep type="groovy" name="VerifyMessage-NhinOutbound-RespondingGateway(2.2)" disabled="true">
+      <con:testStep type="groovy" name="VerifyMessage-NhinOutbound-RespondingGateway(2.2)">
```
Comment (TabassumJafri) [inline Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml:6882]: Not sure why this step was enabled

File: Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml (hunk 1 above)
Comment (sovanncgi) [inline Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml:6315]: This step was previously failing so it was disabled. Fix to test now allows test to pass so this step has been enabled again.

File: Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml (hunk 2 above)
Comment (sovanncgi) [inline Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/AuditLogging-Standard-soapui-project.xml:6882]: This step was previously failing so it was disabled. Fix to test now allows test to pass so this step has been enabled again.

**Comment by TabassumJafri** [comment]
Reviewed the PR and good with merge

**Comment by christophermay07** [comment]
Question -- you have several added imports that you don't use.  Can these be removed? (if so, make sure you have our NetBeans formatter imported, and that the option to remove unused imports on save is checked).

**Pull request merged**